src/software/ai/hl/stp/tactic/receiver_tactic.cpp

```diff
@@ -100,10 +101,10 @@ void ReceiverTactic::calculateNextAction(ActionCoroutine::push_type& yield)
             Point ideal_position    = shot.getPointToShootAt();
             Angle ideal_orientation = shot.getOpenAngle();
 
-            move_action->updateControlParams(*robot, ideal_position, ideal_orientation, 0,
-                                             DribblerMode::OFF, AutochickType::AUTOKICK,
-                                             BallCollisionType::ALLOW);
-            yield(move_action);
+            autokick_move_action->updateControlParams(
+                *robot, ideal_position, ideal_orientation, 0, DribblerMode::OFF,
+                BALL_MAX_SPEED_METERS_PER_SECOND - 1, BallCollisionType::ALLOW);
```

Comment:
What's the status of subtracting 1?

Reply:
We don't want to fly too close to the sun so to speak. If the ball speed calibration is a bit off then we could break ssl rules.

Reply:
Could we add a comment or something to indicate this so it seems less of a 'magic number' so to speak?